bench/irmin-pack/tree.ml

```diff
@@ -1233,5 +1233,22 @@ let main_term =
     $ results_dir)
 
 let () =
-  let info = Term.info "Benchmarks for tree operations" in
+  let man =
+    [
+      `S "DESCRIPTION";
+      `P
+        "Benchmarks for tree operations. Requires traces of operations, \
+         download them (`wget trace.repr`) from: ";
+      `P
+        "Trace with $(b,10310) commits \
+         http://data.tarides.com/irmin/data4_10310commits.repr";
+      `P
+        "Trace with $(b,100066) commits \
+         http://data.tarides.com/irmin/data4_100066commits.repr";
+      `P
+        "Trace with $(b,654941) commits \
+         http://data.tarides.com/irmin/data4_654941commits.repr";
+    ]
```

Comment:
would be nice to point to the trace format specification too (when this spec exists :p)

Reply:
Soooon